app/models/timeslot.rb

```diff
@@ -3,5 +3,5 @@ class Timeslot < ApplicationRecord
   validates :start_time, presence: true
 
   belongs_to :event
-  has_many :preferences, through: :event, class_name: 'Event'
+  has_many :preferences, through: :event, class_name: "Event"
```

Comment:
I thought we were preferring single quotes when double quotes are not necessary

Reply:
We are, but I changed this when I made that first push. I'm not sure how it got back in there.